src/component.vue

```diff
@@ -80,7 +70,10 @@ export default {
      * The input's value
      * @model
      */
-    value: [String, Number]
+    modelValue: [String, Number],
+    modelModifiers: {
```

Comment:
note: built in modifiers like [lazy](https://vuejs.org/guide/essentials/forms.html#lazy) do not work on the custom components. Here we can use it like a [custom modifier](https://vuejs.org/guide/components/events.html#usage-with-v-model)